src/chunk.c

```diff
@@ -4937,3 +5149,226 @@ ts_chunk_get_osm_slice_id(int32 chunk_id, int32 time_dim_id)
 	const int slice_id = ds->fd.id;
 	return slice_id;
 }
+
+/*
+ * Initialization and access method for ChunkVec. This needs to be extended to support additional
+ * operations.
+ */
+static ChunkVec *
+chunk_vec_expand(ChunkVec *chunks, uint32 new_capacity)
+{
+	if (chunks != NULL && chunks->capacity > new_capacity)
+		return chunks;
```

Comment:
This `return chunks` typically will never be hit in code coverage because of the way the callers of this function check for the existing capacity before calling expand with an incremented limit.